src/transaction.rs

```diff
@@ -29,9 +29,32 @@ pub use wasm_bindgen::prelude::*;
 mod test;
 
 static PROVER: Lazy<LocalTxProver> = Lazy::new(|| {
-    LocalTxProver::from_bytes(&[], &[]) // TODO: add params
+    let (sapling_spend_bytes, sapling_output_bytes) = fetch_params().expect("Cannot fetch params");
+    LocalTxProver::from_bytes(&sapling_spend_bytes, &sapling_output_bytes)
 });
 
+fn fetch_params() -> Result<(Vec<u8>, Vec<u8>), Box<dyn Error>> {
+    let c = reqwest::blocking::Client::new();
+    let sapling_output_bytes = c
+        .get("https://duddino.com/sapling-output.params")
+        .send()?
+        .bytes()?;
+    let sapling_spend_bytes = c
+        .get("https://duddino.com/sapling-spend.params")
+        .send()?
+        .bytes()?;
+
+    assert_eq!(
```

Comment:
Instead of this, return an Err

Reply:
right this was originally a test and I forgot to change it